docs/01-start-here/01-installation-steps.md

```diff
@@ -97,12 +97,6 @@ Mac:
 ```bash
 brew install node@6
 ```
-
-#### Grunt (build tool)
-
-Ubuntu/Mac:
-```
-npm -g install grunt-cli
 ```
```

Comment:
Need to kill these backticks or the formatting of the markdown file gets messed up